client/galaxy/scripts/mvc/tool/tool-form-composite.js

```diff
@@ -446,23 +446,8 @@ define([ 'utils/utils', 'utils/deferred', 'mvc/ui/ui-misc', 'mvc/form/form-view'
         /** Templates */
         _templateSuccess: function( response ) {
             if ( response && response.length > 0 ) {
-                var $message = $( '<div/>' ).addClass( 'donemessagelarge' )
-                                .append( $( '<p/>' ).text( 'Successfully ran workflow \'' + this.model.get( 'name' ) + '\'. The following datasets have been added to the queue:' ) );
-                for ( var i in response ) {
-                    var invocation = response[ i ];
-                    var $invocation = $( '<div/>' ).addClass( 'workflow-invocation-complete' );
-                    invocation.history && $invocation.append( $( '<p/>' ).text( 'These datasets will appear in a new history: ' )
-                                                     .append( $( '<a/>' ).addClass( 'new-history-link' )
-                                                                         .attr( 'data-history-id', invocation.history.id )
-                                                                         .attr( 'target', '_top' )
-                                                                         .attr( 'href', '/history/switch_to_history?hist_id=' + invocation.history.id )
-                                                                         .text( invocation.history.name ) ) );
-                    _.each( invocation.outputs, function( output ) {
-                        $invocation.append( $( '<div/>' ).addClass( 'messagerow' ).html( '<b>' + output.hid + '</b>: ' + output.name ) );
-                    });
-                    $message.append( $invocation );
-                }
-                return $message;
+                return $( '<div/>' ).addClass( 'donemessagelarge' )
+                                    .append( $( '<p/>' ).text( 'The requested workflows have been queued and datasets will appear as jobs are created - you will need to refresh your history panel to see these.' ) );
```

Comment:
I would have preferred to leave the dataset details in the success message but the existing api endpoint we are planning to use does not provide that information, nor does it allow to submit the datasets to a new history or supports batch modes. However, it should be possible to enhance the existing endpoint and add the latter two features and then I will add the new_history_link back to the success message.